pkgs/applications/office/beancount/beancount-ing-diba.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+, poetry
+}:
+
+python3Packages.buildPythonApplication rec {
```

Depends on my suggestion below.
```suggestion
, fetchFromGitHub
, buildPythonApplication
, poetry
}:

buildPythonApplication rec {
```